demo-terraform/aws/serverless-app/lambdas.tf

```diff
@@ -34,43 +34,52 @@ resource "aws_iam_role" "lambdas" {
 }
 EOF
 
-    tags = local.project_tags
+  tags = local.project_tags
 
 }
 
 ## Search basic policy for lambdas
 data "aws_iam_policy" "basic_for_lambdas" {
-    name = "AWSLambdaBasicExecutionRole"
+  name = "AWSLambdaBasicExecutionRole"
 }
 resource "aws_iam_role_policy_attachment" "basic_for_lambdas" {
-    for_each = aws_iam_role.lambdas
+  for_each = aws_iam_role.lambdas
 
-    role = each.value.name
-    policy_arn = data.aws_iam_policy.basic_for_lambdas.arn
+  role       = each.value.name
+  policy_arn = data.aws_iam_policy.basic_for_lambdas.arn
 }
 
 resource "aws_lambda_function" "lambda" {
```

Comment:
**Accurics remediation**
 
**[LOW]: Ensure tracing is enabled for AWS Lambda Functions**
 
**Vulnerability**
Lambda tracing if not enabled can have an impact the integrity and incident response.
 
**Remediation**
In AWS Console - 
 1. Sign in to AWS Console and go to the Functions page on the Lambda console.
 2. Select a function.
 3. Select Configuration and then choose Monitoring tools.
 4. Select Edit.
 5. Under X-Ray, enable Active tracing.
 6. Click Save.

source: accurics-